sequencing/vcf_pca/Readme.md

```diff
@@ -1,21 +1,18 @@
-<!-- dx-header -->
 # Calculate PCs from VCF (DNAnexus Platform App)
 
-Uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF or PLINK bed/bim/fam files.
+## What does this applet do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF (`*.vcf.gz`) or PLINK files(`*.bed`, `*.bim`, `*.fam`). The app can process both type of data at the same time.
```

Comment:
"files(`*.bed`, `*.bim`, `*.fam`)" --> "files (`*.bed`, `*.bim`, `*.fam`)"

"both type of data" --> "both types of data"